core/src/main/java/org/geysermc/geyser/level/physics/CollisionManager.java

```diff
@@ -293,10 +292,10 @@ public boolean correctPlayerPosition() {
         return true;
     }
 
-    public Vector3d correctPlayerMovement(Vector3d movement, boolean checkWorld, boolean teleported) {
+    public Vector3d correctPlayerMovement(Vector3d movement, boolean checkWorld) {
         // On the teleported check: see https://github.com/GeyserMC/Geyser/issues/2540
```

Comment:
the "teleported" boolean is replaced with "checkWorld", right? I'd suggest renaming the variable in the comment too